ci-operator/step-registry/addon/install/addon-install-commands.sh

```diff
@@ -7,15 +7,33 @@ set -o verbose
 
 CLUSTER_NAME=$(cat "${SHARED_DIR}/cluster-name")
 OCM_TOKEN=$(cat /var/run/secrets/ci.openshift.io/cluster-profile/ocm-token)
+BREW_TOKEN=$(cat /var/run/secrets/ci.openshift.io/cluster-profile/brew-token)
+RUN_COMMAND="
+    --cluster ${CLUSTER_NAME} \
+    --token ${OCM_TOKEN} \
+    --api-host ${API_HOST} \
+    --timeout ${TIMEOUT} \
+    --brew-token ${BREW_TOKEN} \
+    "
+
+ADDONS_CMD=""
+for i in {1..4}; do
+  ADDON_VALUE=$(eval "echo $"ADDON$i"_CONFIG")
+  if [[ -n $ADDON_VALUE ]]; then
+    ADDONS_CMD="${ADDONS_CMD} --addons ${ADDON_VALUE}"
+  fi
+done
+
+echo "$ADDONS_CMD"
+
+RUN_COMMAND="${RUN_COMMAND} ${ADDONS_CMD}"
+
+if [ -n "${PARALLEL}" ]; then
+    RUN_COMMAND="${RUN_COMMAND} --parallel"
+fi
+
```

Comment:
If `rosa-cli` is not the case for **all** scenarios in `openshift-ci`:

```suggestion

if [ -n "${ROSA}" ]; then
    RUN_COMMAND="${RUN_COMMAND} --rosa"
fi
```